dailystats.py

```diff
@@ -10,13 +10,26 @@
 import csv
 from datetime import datetime
 import logging
-from sys import stderr, exit as sys_exit
+from sys import stderr, stdout, exit as sys_exit
 from typing import List, Tuple
 
 
-LOGGER = logging.getLogger(__name__)
+def configure_logger(debug: bool = False):
+    """Configure logs to stdout."""
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.DEBUG)
+    handler = logging.StreamHandler(stdout)
+    handler.setLevel(logging.DEBUG if debug else logging.INFO)
+    formatter = logging.Formatter('%(message)s')
+
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+    return logger

```

Comment:
I forgot I needed to set all of these up earlier to send the debug and info logs to stdout